src/app/histogram/histogram.component.css

```diff
@@ -9,24 +9,32 @@
 }
 
 .step {
-  font-size: 6px;
+  font-size: 8px;
   width: 20px;
-  height: 10px;
+  height: 18px;
   position: absolute;
+  /* stylelint-disable-next-line length-zero-no-unit */
   right: 0px;
-  margin: 2px;
+  margin: 1px;
   z-index: 100;
   vertical-align: middle;
   line-height: 3px;
-  border-radius: revert;
+  border-bottom-right-radius: 0.25rem;
```

Comment:
why rem?

Reply:
because the input box has border-radius in rem